tests/template_test.go

```diff
@@ -84,7 +84,7 @@ func TestTemplates(t *testing.T) {
 		template := template
 		templateName := template.Name
 
-		if isBlackListedTest(templateName, blackListed) {
+		if isBlackListedTest(templateName, blackListed) || templateName != "java" {
```

Comment:
you shouldn't need the `||`, we don't need it for go

Reply:
Not ready for review yet - I"m messing with it to only run java tests to speed up iterating on CI. I'll remove when it passes.